src/esm.ts

```diff
@@ -94,6 +94,17 @@ export function genExport(
   return _genStatement("export", specifier, exports, options);
 }
 
+/**
+ * Generate an ESM `export default` statement.
+ *
+ * @group ESM
+ */
```

Comment:
```suggestion
/**
 * Generate an ESM `export default` statement.
 * 
 * @example
 * ```js
 * genExportDefault("obj");
 * // ~> `export default obj;`
 * 
 * genExportDefault("{ foo, bar }");
 * // ~> `export default { foo, bar };`
 * ```
 *
 * @group ESM
 */
```